pyproject.toml

```diff
@@ -85,9 +85,6 @@ auth-tool = "asreview.webapp.entry_points.auth_tool:AuthTool"
 simulate = "asreview.simulation.cli:_cli_simulate"
 algorithms = "asreview.models.cli:cli_algorithms"
 
-[project.entry-points."asreview.entry_points_internal"]
-web_run_model = "asreview.webapp.entry_points.run_model:main"
-
```

Comment:
Add the task manager here.

Reply:
I've kept asreview.entry_points_internal since it provides a place where more of these "hidden" entry-points can exist. We never know what the future may bring. I like the generic name over a specific name.